common/lib/xmodule/xmodule/modulestore/xml_exporter.py

```diff
@@ -113,7 +113,7 @@ def export_to_xml(modulestore, contentstore, course_key, root_dir, course_dir):
         export_extra_content(export_fs, modulestore, course_key, xml_centric_course_key, 'about', 'about', '.html')
 
         # export the grading policy
-        course_run_policy_dir = policies_dir.makeopendir(course.location.name)
+        course_run_policy_dir = policies_dir.makeopendir(course.location.run)
```

Comment:
Without this change, the policy file was stored in a directory called "course" instead of the run name for split modulestore only (there was a test for this).